build/archives/BUILD.gn

```
@@ -5,6 +5,20 @@
 import("//flutter/build/zip_bundle.gni")
 import("//flutter/common/config.gni")
 
+generated_file("artifacts_entitlement_config") {
+  outputs = [ "$target_gen_dir/entitlements.txt" ]
+  if (flutter_runtime_mode == "debug") {
```

In case we need to add more default artifacts, it's cleaner to remove the else
```suggestion
  contents = [ "gen_snapshot" ]
  if (flutter_runtime_mode == "debug") {
    contents += [
      "flutter_tester",
      "libtessellator.dylib",
      "impellerc",
    ]
  }
```

---

Sure, removed.